dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
>I don't know why something like this is not widely adopted

My reaction is exactly why and honestly this is the first time I encounter this kind of shorthand...